vlc-android/src/org/videolan/vlc/gui/browser/FilePickerFragment.kt

```diff
@@ -112,7 +112,7 @@ class FilePickerFragment : FileBrowserFragment() {
 
     override fun defineIsRoot() = mrl?.run {
         if (startsWith("file")) {
-            val path = Strings.removeFileProtocole(this@run)
+            val path = this@run.removeFileProtocole()
```

Comment:
do you need `this@run.` here?

Maybe calling `removeFileProtocole()` directly would work.